DEVGUIDE.md

```diff
@@ -47,7 +46,9 @@ At minimum, you need to run `.\build RunGeneratorForFabulousXamarinForms` to ens
 ## Dev Notes - Running the generator
 
 The Generator is built and run as part of the default build command
-If you only want to build the tools and run the generator, use the following commands:
+If you only want to build the tools and run the generator, use the following commands.
+
+### Please run the generator if there are build errors related to "Missing Xamarin.Forms.Core.fs".
```

Comment:
Could you just set it in bold and not make it a title, please?
Thanks

Reply:
Done